server/chat-plugins/mafia.ts

```diff
@@ -2106,7 +2104,7 @@ export const pages: Chat.PageTable = {
 				buf += `: <button class="button" name="send" value="/msgroom ${room.roomid},/mafia revive ${eliminated.id}">Revive</button></p>`;
 			}
 			buf += `<hr/></details></p>`;
-			if (game.dayNum > 1) {
+			if (game.phase === "day" || game.dayNum > 0) {
```

Comment:
```suggestion
			if (game.dayNum > 0) {
```
See above suggestions for reasoning.